<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imgXKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgYKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgRKey">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgGKey">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgBKey">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgXVal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgYVal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgRVal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgGVal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgBVal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="histogramPushButton">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>12</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalizerPushButton">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>40</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Equalizer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brightenPushButton">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>100</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="normalizePushButton">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otsuBinarizePushButton">
    <property name="geometry">
     <rect>
      <x>666</x>
      <y>130</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Otsu Binarization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="niblackBinarizepushButton">
    <property name="geometry">
     <rect>
      <x>667</x>
      <y>160</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Niblack Binarization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thresholdBinarizepushButton">
    <property name="geometry">
     <rect>
      <x>667</x>
      <y>190</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold Binarization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convolutePushButton">
    <property name="geometry">
     <rect>
      <x>667</x>
      <y>220</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convolution Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kuwaharaPushButton">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>250</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kuwahara Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="medianFilterPushButton">
    <property name="geometry">
     <rect>
      <x>663</x>
      <y>280</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imageOpsPushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Operations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayScalePushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GrayScale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="customConvKernelPushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom Conv Kernel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
